README.md

```diff
@@ -45,18 +44,97 @@ Check out our live demo on [demo.pyrra.dev](https://demo.pyrra.dev)!
 
 Feel free to give it a try there!
 
-## Installation
+## How It Works
+
+There are three components of Pyrra, all of which work through a single binary:
+
+- The frontend which displays SLOs, error budgets, burn rates, etc.
+- The api which delivers information about SLOs from a backend (like Kubernetes)
+  to the frontend.
+- A reconciler which watches for new SLO objects and then creates Prometheus 
+  recording rules for each.
+
+For the reconciler to do its magic, an SLO object needs to provided in 
```

Comment:
```suggestion
For the backend/operator to do its magic, an SLO object needs to provide in 
```

Reply:
I think this should be `needs to be provided` 😬

Reply:
I've changed it, PTAL!